.github/workflows/zos-build.yml

```diff
@@ -28,7 +28,7 @@ jobs:
   build-server:
     runs-on: ubuntu-latest
     concurrency:
-      group: ${{ github.workflow }}
+      group: "zos-build"
```

Comment:
please not that I'mm using the same group name to avoid conflicts between the native/c and pythong builds 😋